xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEFilter.cpp

```diff
@@ -277,6 +280,8 @@ int CActiveAEFilter::ProcessFilter(uint8_t **dst_buffer, int dst_samples, uint8_
       return -1;
     }
 
+    m_SamplesOut = m_pOutFrame->pts;
```

Comment:
Yes, I added this logging:
http://paste.ubuntu.com/24450547/
and got this log:
http://paste.ubuntu.com/24450549/
Certainly looks like pts=AV_NOPTS_VALUE